marketaccess/forms.py

```diff
@@ -2,99 +2,197 @@
 from directory_components import forms, fields, widgets
 from django.utils.safestring import mark_safe
 
-from django.forms import Select, Textarea
+from django.forms import Select, Textarea, TextInput
+
+
+class CurrentStatusForm(forms.Form):
+    error_css_class = 'input-field-container has-error'
+    STATUS_CHOICES = (
+        (
+            1,
+            'My perishable goods or livestock are blocked in transit'
+        ), (
+            2,
+            'I’m at immediate risk of missing a commercial opportunity'
+        ), (
+            3,
+            'I’m at immediate risk of not fulfilling a contract'
+        ), (
+            4,
+            'I need resolution quickly, but  I’m not at immediate risk of loss'
+        ),
+    )
+
+    status = fields.ChoiceField(
+        label='Select which option best applies to you',
+        widget=widgets.RadioSelect(
+            use_nice_ids=True, attrs={'id': 'radio-one'}
+        ),
+        choices=(
+            (choice_code, choice) for choice_code, choice in STATUS_CHOICES
+        )
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(CurrentStatusForm, self).__init__(*args, **kwargs)
+
+        self.fields['status'].error_messages = {
```

Comment:
as far as I ca tell there no need to do it in `__init__` - this can be done on the field definition